bin/canary

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+require 'fileutils'
+
+# Path to the application root.
+APP_ROOT = File.expand_path('..', __dir__)
+
+# We want to guard against failures at each stage of this
+def system!(*args)
+  system(*args) || abort("\n== Command #{args} failed ==")
+end
+
+require "bundler/setup"
+$LOAD_PATH.unshift File.expand_path("../../lib", __FILE__)
+
+require "tooling_invoker"
+
+puts "Checking Canary..."
+
+unless ToolingInvoker::CheckCanary.()
+  `sudo shutdown now`
+end
```

Comment:
This shuts down the entire machine, right?

Reply:
Yep